src/Agent.Listener/Configuration/LinuxServiceControlManager.cs

```diff
@@ -100,37 +100,31 @@ public override void StartService(string serviceName)
                 ChangeOwnershipToLoginUser(filesToChange);
                 ReloadSystemd();
                 ExecuteSystemdCommand("start " + serviceName);
+                _term.WriteLine(StringUtil.Loc("ServiceStartedSuccessfully", serviceName));
             }
             catch (Exception)
             {
-                _term.WriteError(StringUtil.Loc("LinuxServiceStartFailed"));
+                _term.WriteError(StringUtil.Loc("CanNotStartService"));
                 throw;
             }
         }
 
-        protected virtual string GetUnitFile(string serviceName)
+        public override void StopService(string serviceName)
         {
-            return Path.Combine(SystemdPathPrefix, serviceName);
-        }
-
-        protected virtual bool CheckIfSystemdExists()
-        {
-            Trace.Info(nameof(CheckIfSystemdExists));
+            Trace.Info(nameof(StopService));
```

Comment:
...I'm uncertain whether Entering meets the bar for info (my gut is that it does not). I would suggest keep Entering as Verbose.